GTScript-Parallel-model.md

```diff
@@ -103,3 +103,107 @@ for k in range(start, end):
         u[i, j, k] = 1
     parfor [i_start:i_end, j_start:j_end]:
         b[i, j, k] = u[i-2,j,k] + u[i+1,j,k] + u[i,j-1,k] + u[i,j-2,k]
+```
+
+## Conditionals
+
+### If-Statements
+
+- Conditions in `if` statements are required to be of scalar type, e.g. scalar variables (not fields) or scalar expressions (no fields involved).
+- Each statement inside the if and else branches is executed as a `parfor` loop (same as statements outside of branches).
```

Comment:
Writing to a (non-temporary?) field and then reading from that field at a horizontal offset is not allowed. Not sure if we need to specify this here, but we should for sure have a check in gt4py that makes sure this is adhered to.

Reply:
It is allowed (but currently not working, that's the problem we discussed on Tuesday).